### DISCUSSION TITLE
Thinkin' out loud

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1176] [Q&A]
I created and maintain an H&I site and our AA Convention site. Can the meeting app be used for panels and convention schedules? Any suggestions? I have it working but just looking at optimum solutions. 

d

**Comment by kiyote33** [comment]
Hi d,
Our 12 Step Meeting List plugin is designed for recovery organizations to display weekly scheduled meetings on their websites. Included in its many features is the ability to generate JSON which can be imported by GSO's Meeting Guide app. That's how the majority of sites use it, particularly those in the AA community. That being said, I'm sure there are lot's of creative types out there that may have devised other uses for it. It is free, open source code which you can download and tweak to your heart's content. Hope I've answered your question...

**Reply by dougie-wougie** [reply]
Thank you. I am using it for its intended purpose for our Central Office but I wanted to ask if this community was doing more with or without the 12 Step Meeting plugin to do other things such as H & I panels and scheduling. I have been using time.ly and some table plugins, but I figured this community is most likely doing smarter things than what I have been using. I created and maintain four AA-related websites and as my sites reach further, I want to do the best I can for each of them. 

D.

**Reply by joshreisner** [reply]
thanks for the vote of confidence! we would not recommend TSML for other uses than listing weekly recovery meetings but like @kiyote33 noted, it's open source software so developers are welcome to adapt it to other purposes

i hear good things about [the events calendar](https://theeventscalendar.com/) and i know other event management systems have been discussed on the tiaa forum. if it were between TSML and time.ly i would choose time.ly 👍
